### PR TITLE
[17.0][MIG] membership_initial_fee: Migration to 17.0

### PR DESCRIPTION
**Pull request by RogerSans**
Standard migration
T-6474

@HaraldPanten 
@ValentinVinagre

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]
Functional review. LGTM

**Comment by HaraldPanten** [comment]
@pedrobaeza do you mind assigning the migration to us? THX in advance.

File: membership_initial_fee/models/account_move_line.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Copyright 2015 Tecnativa - Pedro M. Baeza
+# Copyright 2017 Tecnativa - David Vidal
+# Copyright 2015-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2019 Onestein - Andrea Stirpe
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0
+
+from odoo import api, models
+from odoo.tools.translate import _
```
Comment (manuelregidor) [inline membership_initial_fee/models/account_move_line.py]: Could you import "_" in the previous import line?

File: membership_initial_fee/models/account_move_line.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                ("account_invoice_line", "not in", (self.id,)),
+                ("state", "not in", ["none", "canceled"]),
+            ]
+        )
+        return not bool(member_lines)
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        move_lines = super().create(vals_list)
+        for move_line in move_lines:
```
Comment (manuelregidor) [inline membership_initial_fee/models/account_move_line.py]: You could include the "if" condition in this line using "filtered". Not a blocking comment though.

**Review by manuelregidor — CHANGES_REQUESTED** [review]

File: membership_initial_fee/models/account_move_line.py (hunk 1 above)
Comment (RogerSans) [reply membership_initial_fee/models/account_move_line.py]: Done

File: membership_initial_fee/models/account_move_line.py (hunk 2 above)
Comment (RogerSans) [reply membership_initial_fee/models/account_move_line.py]: Done, can you check pls?

**Review by manuelregidor — APPROVED** [review]
Technical review. LGTM

**Comment by HaraldPanten** [comment]
@pedrobaeza could we merge?

THX

**Comment by pedrobaeza** [comment]
The migration is pretty straight-forward following the migration guide, so having 2 new contributors seems a bit excessive IMO.

**Comment by HaraldPanten** [comment]
@RogerSans could you remove Valentín from contributors, please?

THX

**Comment by RogerSans** [comment]
Done THX

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**